tests/zfs-tests/tests/functional/rsend/send_realloc_files.ksh

```diff
@@ -71,7 +71,7 @@ for i in {1..$passes}; do
 	# Randomly modify several dataset properties in order to generate
 	# more interesting incremental send streams.
 	rand_set_prop $POOL/fs checksum "off" "fletcher4" "sha256"
-	rand_set_prop $POOL/fs compression "off" "lzjb" "gzip" "lz4"
+	rand_set_prop $POOL/fs compression "off" "lzjb" "gzip" "lz4" "zstd"
```

Comment:
Fix for this is underway...
Forgot it during my refactor of testing code... See #9679

Reply:
Fix for this is merged, you can Rebase and get rid of this file now! :)